wdl-lint/src/rules/version_formatting.rs

```diff
@@ -40,21 +41,21 @@ fn expected_blank_line_after_version(span: Span) -> Diagnostic {
 
 /// Creates a diagnostic for unexpected whitespace before the version statement.
 fn whitespace_before_version(span: Span) -> Diagnostic {
-    Diagnostic::error("unexpected whitespace before the version statement")
+    Diagnostic::note("unexpected whitespace before the version statement")
         .with_rule(ID)
         .with_highlight(span)
 }
 
 /// Creates a diagnostic for a comment inside the version statement.
 fn comment_inside_version(span: Span) -> Diagnostic {
-    Diagnostic::error("unexpected comment inside the version statement")
+    Diagnostic::note("unexpected comment inside the version statement")
```

Comment:
I think this can stay an error. I think that's a reasonable position to take

Reply:
I don't think that jives with the definitions we now have [here](https://github.com/stjude-rust-labs/wdl/blob/main/CONTRIBUTING.md#whats-the-difference-between-error-warning-and-note)

The comment doesn't interrupt parsing/validation/meaning/etc, so `error` would be wrong. I suppose `warning` could fit as it's potentially confusing? But IMO it really just boils down to style (i.e. a `note`), although a particularly egregious example of bad style.